### PR TITLE
Rename typography folder to text

### PR DESCRIPTION
**Pull request by joshwooding**
After some discussions with @jeanlaleuf he suggested the typography folder/stories should be renamed to Text.

Plus this fixes an issue with #380

### CODE REVIEW

File: packages/lab/stories/typography/BaseText.stories.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { Text, TextProps } from "@jpmorganchase/uitk-lab";
 import { useEffect, useState } from "react";
```
Comment (libertymayc) [inline packages/lab/stories/typography/BaseText.stories.tsx:4]: Should the stories folder also change to /text?

Comment (libertymayc) [reply packages/lab/stories/typography/BaseText.stories.tsx:4]: also does this need a changeset?

Comment (alycrys) [reply packages/lab/stories/typography/BaseText.stories.tsx:4]: Also the e2e tests folder

Comment (joshwooding) [reply packages/lab/stories/typography/BaseText.stories.tsx:4]: Whoops, I made the change locally but it wasn't picked up...

I don't think it needs a changeset as it's not changing any user facing code. Only refactoring the structure and we don't add changelogs for docs changes at the moment.

**Review by libertymayc — APPROVED** [review]

**Review by alycrys — APPROVED** [review]

**Pull request merged**